modules/ROOT/examples/live-demos/url-dialog/example.js

```diff
@@ -2,17 +2,15 @@ tinymce.init({
   selector: 'textarea#url-dialog',
   toolbar: 'urldialog',
   height: 300,
-  setup: function (editor) {
+  setup: (editor) => {
     editor.ui.registry.addButton('urldialog', {
       icon: 'code-sample',
-      onAction: function () {
-        editor.windowManager.openUrl({
-          title: 'URL Dialog Demo',
-          url: '../../demo/url-dialog-demo/external-page.html',
-          height: 640,
-          width: 640
-        });
-      }
+      onAction: () => editor.windowManager.openUrl({
+        title: 'URL Dialog Demo',
+        url: 'external-page.html',
```

Comment:
Confirm this 
url: 'external-page.html',
is different from the one below
url: '{{attachmentsdir}}/url-dialog-demo/external-page.html',